src/Wallabag/CoreBundle/Resources/translations/messages.it.yml

```diff
@@ -295,43 +297,43 @@ howto:
-        # all_pages_title: Shortcuts available in all pages
-        # go_unread: Go to unread
-        # go_starred: Go to starred
-        # go_archive: Go to archive
-        # go_all: Go to all entries
-        # go_tags: Go to tags
-        # go_config: Go to config
-        # go_import: Go to import
-        # go_developers: Go to developers
-        # go_howto: Go to howto (this page!)
-        # go_logout: Logout
-        # list_title: Shortcuts available in listing pages
-        # search: Display the search form
-        # article_title: Shortcuts available in entry view
-        # open_original: Open original URL of the entry
-        # toggle_favorite: Toggle star status for the entry
-        # toggle_archive: Toggle read status for the entry
-        # delete: Delete the entry
-        # material_title: Shortcuts available with Material theme only
-        # add_link: Add a new link
-        # hide_form: Hide the current form (search or new link)
-        # arrows_navigation: Navigate through articles
-        # open_article: Display the selected entry
+        page_description: Ecco le scorciatoie disponibili su wallabag.
+        shortcut: Scorciatoia
+        action: Azione
+        all_pages_title: Scorciatoie disponibili in tutte le pagine
+        go_unread: Vai ai non letti
+        go_starred: Vai ai preferiti
+        go_archive: Vai in archivio
+        go_all: Vai a tutti gli articoli
+        go_tags: Vai alle etichette
+        go_config: Vai su configurazione
+        go_import: Vai su importazione
+        go_developers: Vai su sviluppatori
+        go_howto: Vai su Come fare (questa pagina!)
+        go_logout: Disconnetti
+        list_title: Scorciatoie disponibili quando si elencano le pagine
+        search: Mostra il modulo di ricerca
+        article_title: Scorciatoie disponibili in vista articoli
+        open_original: Apri URL originale dell'articolo
+        toggle_favorite: Cambia stato Preferito dell'articolo
+        toggle_archive: Cambia stato Letto dell'articolo in letto
+        delete: Cancella l'articolo
+        material_title: Scorciatoie disponibili solo con il tema Material
+        add_link: Aggiungi un nuovo link
+        hide_form: Nascondi il modulo corrente (ricerca o nuovo link)
+        arrows_navigation: Naviga tra gli articoli
+        open_article: Mostra l'articolo selezionato
 
 quickstart:
     page_title: 'Introduzione'
-    # more: 'More…'
+    more: 'Più…'
     intro:
         title: 'Benvenuto su wallabag!'
         paragraph_1: "Un tour in cui ti guideremo per scoprire e che ti mostrerà delle funzionalità che potrebbero interessarti."
         paragraph_2: 'Seguici!'
     configure:
         title: "Configura l'applicazione"
-        # description: 'In order to have an application which suits you, have a look into the configuration of wallabag.'
+        description: 'Per avere un'applicazione che ti soddisfi, dai un'occhiata alla configurazione di wallabag.'
```

Comment:
I don't know who did that but we should put double only when it's required I think